.github/workflows/cargo-audit-pr.yml

```
@@ -0,0 +1,43 @@
+name: Cargo Audit (PR gate)
+
+# Blocks PRs that introduce dependencies with known advisories. The daily
+# audit workflow (.github/workflows/audit.yml) still catches advisories
+# published after merge; this job closes the window between merge and the
+# next scheduled audit tick.
+
+on:
+    pull_request:
+        branches: [ main ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)